addons/redis/templates/componentdefinition-redis-sentinel.yaml

```diff
@@ -76,6 +76,30 @@ spec:
           name: default
           optional: false
           password: Required
+    ## the default password of redis connection
+    - name: REDIS_DEFAULT_PASSWORD
+      valueFrom:
+        credentialVarRef:
+          ## reference the current component definition name
+          compDef: {{ include "redis.componentDefName" . }}
+          name: default
+          optional: true
+          password: Optional
+          multipleClusterObjectOption:
+            strategy: individual
+    - name: SENTINEL_POD_NAME_LIST
+      valueFrom:
+        componentVarRef:
+          compDef: {{ include "redis-sentinel.componentDefName" . }}
+          optional: true
+          instanceNames: Optional
+    - name: SENTINEL_HEADLESS_SERVICE_NAME
```

Comment:
It is recommended to directly use SENTINEL_POD_FQDN_LIST（which refers from componentVarRef.podFqdns） instead of SENTINEL_POD_NAME_LIST and SENTINEL_HEADLESS_SERVICE_NAME.